content/en/hosting/cht/migration/preparing-for-5.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Preparing to upgrade to CHT 5.0"
+linkTitle: "Preparing for CHT 5.0"
+weight: 
+description: >
+  Steps to ensure your CHT App will run smoothly on CHT 5.0 and later
+  
+---
+
+## Introduction
+
+CHT releases use [Semantic Versioning](https://en.wikipedia.org/wiki/Semver#Semantic_versioning) (also known as "SemVer"), which means that the CHT upgrade from the major 4.x version to the 5.x version denotes there are breaking changes. The key to a successful upgrade will be to understand and plan for these breaking changes.
+
+While this releases breaking changes mainly apply to Kubernetes based deployments (e.g. Nouveau & Upgrade service), deployments using token login may be affected as well.
+
+As with all upgrades, major or minor, you should always:
+* Have [backups](/hosting/cht/docker/backups/) that have been tested to work
```

Comment:
Do we still need to use the `ref` shortcode for local links?  

```
[backups]({{% ref "hosting/cht/docker/backups" %}})
```

Either way, there are a number of `localhost` links below that I think need fixed too.

Reply:
huh - yeah - you're totally right about using ref and relref!  I thought I remember seeing that hugo updates and hextra didn't require it, but checking our style guide and they're still there! Thanks for noticing!

Fixed use of links and removing localhost in 28c8a806